### PR TITLE
Makefile: Ignore SC2329

### PR DESCRIPTION
**Pull request by vvoland**
See https://github.com/docker/docker-install/pull/413

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -179,7 +179,7 @@ version_gte() {
 	if [ -z "$VERSION" ]; then
 			return 0
 	fi
-	eval version_compare "$VERSION" "$1"
+	version_compare "$VERSION" "$1"
```
Comment (thaJeztah) [inline install.sh:182]: Is there any change in behavior between with / without `eval` ? (LOL, I may have written this code, but I honestly don't recall why it's there 🤔 🙈)

Comment (vvoland) [reply install.sh:182]: It seems to work fine without it:

```
bash-3.2$ export VERSION=23.0
bash-3.2$ version_gte 24.0; echo $?
1
bash-3.2$ version_gte 34.0; echo $?
1
bash-3.2$ version_gte 23.0; echo $?
0
bash-3.2$ version_gte 23.5; echo $?
1
bash-3.2$ version_gte 20.10; echo $?
0
```

Comment (vvoland) [reply install.sh:182]: Btw, the examples in the comment seem wrong 🙈

Comment (vvoland) [reply install.sh:182]: Looks like a mistake in d08fb9eb
```diff
-# VERSION=20.10
+# VERSION=23.0
+# version_gte 23.0  // 0 (success)
```

Comment (thaJeztah) [reply install.sh:182]: > Looks like a mistake in [d08fb9e](https://github.com/docker/docker-install/commit/d08fb9ebee921b2a3654f719b405a5afb0522124)
> 
> ```diff
> -# VERSION=20.10
> +# VERSION=23.0
> +# version_gte 23.0  // 0 (success)
> ```

Isn't that correct though? I.e. 23.0 is "gte" (`>=`) 23.0, or didn't it work?

Comment (thaJeztah) [reply install.sh:182]: Oh! Last line; I see it now; last line had to be updated because 21.10 would now no longer pass

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**